test/framework/options.py

```diff
@@ -3980,6 +3980,60 @@ def test_set_default_module(self):
         else:
             self.assertTrue(False, "Uknown module syntax: %s" % get_module_syntax())
 
+    def test_set_default_module_robot(self):
+        """Test use of --set-default-module --robot."""
+        # create two test easyconfigs, one depending on the other
+        # (using dummy Toolchain easyblock included in the tests)
+        test_ec = os.path.join(self.test_prefix, 'test.eb')
```

Comment:
Won't verify-easyconfig-filenames=True in site easyconfig.cfg (or users) cause problems with that file name for someone running the tests locally?

Reply:
Could be, but if that's so then we need to isolate the tests from the user's configuration.

The tests are assumed to run with the default configuration.

Reply:
ok